spec/api/v2/market_research_library/search_spec.rb

```diff
@@ -134,4 +134,18 @@
       end
     end
   end
+
+  describe 'GET /market_research_library/search.json?size=-1' do
```

Comment:
When testing locally, I only got three results in the JSON:

![screen shot 2015-07-08 at 1 42 58 pm](https://cloud.githubusercontent.com/assets/277504/8563891/56ef840c-2577-11e5-91f3-c116048097ac.png)

The application appeared to fetch all though (I could see the output of the ES query in the foreman output). Can you check this also?